scripts/benchmark/db-benchmark/groupby/G1_1e9_1e2_5_0/q8.py

```diff
@@ -0,0 +1,19 @@
+# Contains code from https://github.com/duckdblabs/db-benchmark/blob/master/pandas/groupby-pandas.py
```

Comment:
Could you clarify what the file name "G1_1e9_1e2_5_0" refers to?

Reply:
There is no link for dataset, it is generated using a script in dbbenchmark.